### PR TITLE
added unit tests for SampleDAO (+1 squashed commit)

### PR DESCRIPTION
**Pull request by przemekk13**
Squashed commits:
[5e6d1fd] added unit tests for  SampleDAO

### CODE REVIEW

File: sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLSampleDAOTest.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
+package uk.ac.bbsrc.tgac.miso.sqlstore;
+
+import com.eaglegenomics.simlims.core.SecurityProfile;
```
Comment (apmasell) [inline sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLSampleDAOTest.java]: Please run organize imports on this file.

Comment (przemekk13) [inline sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLSampleDAOTest.java]: Done

**Comment by djcooke** [comment]
:+1: except :spaghetti:

**Comment by tonydebat** [comment]
Compilation fails with this message:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:testCompile (default-testCompile) on project sqlstore: Compilation failure
[ERROR] /Users/tdebat/code/gsi/miso-lims/sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLSampleDAOTest.java:[56,39] cannot find symbol
[ERROR] symbol: class AbstractDAOTest
```

**Comment by przemekk13** [comment]
Ah one class was moved in develop hence the compile error... try again I committed a fix

**Comment by tonydebat** [comment]
:+1:

**Pull request merged**